manager/integration/tests/test_basic.py

```diff
@@ -3642,6 +3643,127 @@ def test_volume_toomanysnapshots_condition(client, core_api, volume_name):  # NO
                                                "status", "False")
 
 
+@pytest.mark.coretest  # NOQA
+def test_volume_metafile_deleted_empty(client, core_api, volume_name, csi_pv, pvc, pod, pod_make):  # NOQA
+    """
+    Test volume metafile deleted or empty
+
+    Context:
+
+    If the volume still works fine when there is a invalid volume meta file
+    in the replica data path.
+
+    Setup:
+
+    1. Create a pod using Longhorn volume
+    2. Write some data to the volume then get the md5sum
+    3. Create a snapshot
+
+    Steps:
+
+    1. Delete the pod and wait for the volume detached
+    2. Randomly pick up a replica and manually delete or
+       empty the volume meta file in this replica data path
```

Comment:
> Randomly pick up a replica and manually delete or empty the volume meta file in this replica data path
It seems we are testing for 2 cases here. Suggest separating them.

Reply:
Yeah, separate it into two cases `"file does not exist"` and `"file is empty"`. Thanks.